src/styles/index.scss

```diff
@@ -1,3 +1,164 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.card {
+  display: block;
+  justify-content: center;
+  width: 200px;
+  border: 1px solid #f3f3f3;
```

Comment:
Move colors in variables and use them (you need to create `variables.scss` file also)

Reply:
Not resolved